static/src/js/actions/__tests__/urlQuery.test.js

```diff
@@ -206,7 +206,7 @@ describe('updateStore', () => {
               }
             },
             footer: {
-              attributionText: 'NASA Official: Stephen Berrick',
```

Comment:
We don't have to update these I was just doing it for consistency I can go and make these test values or just leave them if people prefer